zulip/zulip/__init__.py

```diff
@@ -486,6 +486,11 @@ def __init__(
 
         self.has_connected = False
 
+        server_settings = self.get_server_settings()
+        self.zulip_version: Optional[str] = server_settings.get("zulip_version")
+        self.feature_level: int = server_settings.get("zulip_feature_level", 0)
+        assert self.zulip_version is not None
```

Comment:
Does `zulip_version` need to use `.get()`?  I think `zulip_version` at least has been present since essentially the beginning of time, and it should probably be an error for it to not be present.